File: src/main/java/econtroller/sensor/Monitor.java
package econtroller.sensor;

import se.sics.kompics.address.Address;
import se.sics.kompics.network.Message;

/**
 * 
 * @author Amir Moulavi
 * @date 2011-04-11
 *
 */

public class Monitor extends Message {

	private static final long serialVersionUID = -4732276884700905212L;

	public Monitor(Address source, Address destination) {
		super(source, destination);
	}

}

File: src/main/java/econtroller/sensor/Monitor.scala
package econtroller.sensor

import se.sics.kompics.network.Message
import se.sics.kompics.address.Address

/**
 * @author Amir Moulavi
 * @date 2011-07-23
 *
 */
class Monitor(source:Address, destination:Address) extends Message(source, destination)